releasenotes/notes/0.25/fix-delay-padding-75937bda37ebc3fd.yaml

```diff
@@ -1,13 +1,12 @@
 ---
 fixes:
   - |
-    Fixed an issue in tranpiler passes for padding delays, which did not respect target's constraints
-    and inserted delays even for qubits not supporting :class:`~.circuit.Delay` instruction.
+    Fixed an issue in tranpiler passes for padding delays, which did not respect target constraints
```

Comment:
This file we can actually remove it was included as part of the 0.24.0 release but after the rc1 in: https://github.com/Qiskit/qiskit-terra/pull/10007